Yosemite/Yosemite/Stores/ProductReview/RetrieveProductReviewFromNoteUseCase.swift

```diff
@@ -9,6 +9,12 @@ public struct ProductReviewFromNoteParcel {
     public let note: Note
     public let review: ProductReview
     public let product: Product
+
+    public init(note: Note, review: ProductReview, product: Product) {
```

Comment:
This `init` I had to declare because I needed to be able to construct this `struct` in [`ReviewsCoordinatorTests`](https://github.com/woocommerce/woocommerce-ios/blob/72d39d7912c631e06faf70e93d0bbe1d53b78b96/WooCommerce/WooCommerceTests/ViewRelated/Notifications/ReviewsCoordinatorTests.swift#L221). It won't compile if there was no `public init` for it. 🤷

Reply:
I wanted to find when a manual constructor is needed and when there's a synthesized one, but haven't seen anything. I like your approach of only making one when there's compile error 😄